### ISSUE TITLE
求Paddleseq训练自己数据集教程

### ISSUE THREAD

**Issue by guoyunqingyue** [PaddlePaddle/PaddleSeg#2876]
### 问题描述 Please describe your issue

按照要求设置了数据集，写好ymal文件，运行之后只显示了设置的信息，也没有报错

**Comment by juncaipeng** [comment]
准备自己数据集文档：https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.7/docs/data/marker/marker_cn.md

建议你进行调试，如果是cpu训练，可能执行比较慢

**Comment by guoyunqingyue** [comment]
我已经按照这个流程制作了数据集，也改写了yml文件，但训练好像还是没有顺利进行

my_unet.yml：
_base_: 'my_unet_eggdata.yml'

batch_size: 2
iters: 10000

model:
  type: UNet
  num_classes: 4
  use_deconv: False
  pretrained: Null

my_unet_eggdata.yml：
batch_size: 2
iters: 10000

train_dataset:
  type: Dataset
  dataset_root: ./datasets/new
  train_path: ./datasets/new/train_list.txt
  transforms:
    - type: ResizeStepScaling
      min_scale_factor: 0.5
      max_scale_factor: 2.0
      scale_step_size: 0.25
    - type: RandomPaddingCrop
      crop_size: [400, 400]
    - type: RandomHorizontalFlip
    - type: RandomDistort
      brightness_range: 0.4
      contrast_range: 0.4
      saturation_range: 0.4
    - type: Normalize
  mode: train

val_dataset:
  type: Dataset
  dataset_root: ./datasets/new
  val_path: ./datasets/new/val_list.txt
  transforms:
    - type: Normalize
  mode: val


optimizer:
  type: sgd
  momentum: 0.9
  weight_decay: 4.0e-5

lr_scheduler:
  type: PolynomialDecay
  learning_rate: 0.01
  end_lr: 0
  power: 0.9

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

**Comment by juncaipeng** [comment]
有更多错误log信息，可以贴出来。如果没有，建议先对基础熟悉，再debug一下。
